<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherApplication</class>
 <widget class="QWidget" name="WeatherApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WeatherApplication</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>321</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="WeatherAddItemTableWidget" name="addItemTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>261</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="todayButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>今天</string>
   </property>
  </widget>
  <widget class="QPushButton" name="futureButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>未来五天</string>
   </property>
  </widget>
  <widget class="QLabel" name="windowTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:16px;color:#FFFFFF;font-weight:bold</string>
   </property>
   <property name="text">
    <string>天气预报</string>
   </property>
  </widget>
  <widget class="QLabel" name="windowIcon">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>2</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/appIcon);</string>
   </property>
  </widget>
  <widget class="QToolButton" name="windowClose">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>5</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="minimization">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>5</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addCityButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加城市</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WeatherAddItemTableWidget</class>
   <extends>QListView</extends>
   <header>weatheradditemtablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
